sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryAsyncClient.java

```diff
@@ -505,6 +506,88 @@ public DirectoryAsyncClient getSubDirectoryClient(String subDirectoryName) {
         return new PagedFlux<>(() -> retriever.apply(null), retriever);
     }
 
+    /**
+     * Closes a handle on a directory or a file at the service. This is intended to be used alongside
+     * {@link #listHandles(Integer, boolean) listHanldes}.
```

Comment:
Typo - but don't bother to even include the different text.

Reply:
Removed